support-workers/deploy-to-code.sh

```diff
@@ -0,0 +1,13 @@
+S3_BUCKET="membership-dist"
```

Comment:
This is a shell script to simplify deploying the zipped artifact to CODE for a quicker testing cycle